tests/clims/models/test_container.py

```diff
@@ -9,9 +9,19 @@
 from clims.services import FloatField, TextField
 from clims.models import Container
 from django.db import IntegrityError
+from tests.fixtures.plugins.gemstones_inc.models import GemstoneSample, GemstoneContainer
+from django.test.utils import override_settings
 
 
 class TestContainer(TestCase):
+    def create_container_with_samples(self, prefix="container", sample_count=10):
+        container = self.create_container(GemstoneContainer, prefix=prefix)
+        for _ in range(sample_count):
+            sample = self.create_substance(GemstoneSample, color='red')
+            container.append(sample)
+        container.save()
+        return container
```

Comment:
This duplicates the code in `ContainerTest`

Reply:
Yes, I'll throw it into the core fixtures.